superset-frontend/src/explore/components/controls/FixedOrMetricControl.jsx

```diff
@@ -107,67 +100,88 @@ export default class FixedOrMetricControl extends React.Component {
     return (
       <div>
         <ControlHeader {...this.props} />
-        <Label onClick={this.toggle}>
-          {this.state.type === controlTypes.fixed && (
-            <span>{this.state.fixedValue}</span>
-          )}
-          {this.state.type === controlTypes.metric && (
-            <span>
-              <span style={{ fontWeight: 'normal' }}>metric: </span>
-              <strong>
-                {this.state.metricValue ? this.state.metricValue.label : null}
-              </strong>
-            </span>
-          )}
-        </Label>
-        <Panel
-          className="panel-spreaded"
-          collapsible
-          expanded={this.state.expanded}
-          onToggle={this.toggle}
+        <Collapse
+          ghost
+          css={css`
+            &.ant-collapse
+              > .ant-collapse-item.ant-collapse-no-arrow
+              > .ant-collapse-header {
+              border: 0px;
+              padding: 0px 0px 8px 0px;
```

Comment:
```suggestion
              padding: 0px 0px 8px 0px;
```
Nit, but would be nice to use `gridUnit`s here and in the `.well` selector below.

Also, I think this deep selector is definitely better than `!important`, but now I'm _just curious_ if we can tack on an `id` to the element itself, and use `&#someID.ant-...` to to throw a little more value at the selector weighting.

Reply:
Applied `gridUnit`.

About the ID the `&` is transformed into a unique class name by emotion giving the appropriate weighting value.